### DISCUSSION TITLE
Introducing malle: a new lib to replace jquery-jeditable

### DISCUSSION THREAD

**Discussion by NicolasCARPi** [NicolasCARPi/jquery_jeditable#242] [Show and tell]
Hello,

So I created [malle](https://github.com/deltablot/malle/), a library written in typescript for doing what jquery-jeditable is doing.

No dependencies, not even jQuery. It is still in the early stages and doesn't implement all the things jquery-jeditable does.

For one, it doesn't do the request for you, it only supports a custom function. This is a design choice to let client code manage the POST request and won't be implemented in the future.

There are also some differences in the options (some are the same). See the [documentation](https://github.com/deltablot/malle/blob/master/DOCUMENTATION.md).

As it is, you can start using it and see if it fits your use case: https://github.com/deltablot/malle/

It will be a really good fit if your project is in typescript because you'll be able to easily import the interfaces/typings. Also if you're trying to slowly move away from having jquery as a dependency, you should definitely give it a try!

Let me know what you think!

**Comment by centdev05** [comment]
Nice initiative. I was looking for something similar, wiil try it
